tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1318,205 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// ScheduleBackupUsingNamespaceLabel perform schedule backup and restore, using namespace label and resource label
+var _ = Describe("{BasicBackupAndScheduleBackupUsingNamespaceLabel}", func() {
+	var (
+		err                        error
+		contexts                   []*scheduler.Context
+		appContexts                []*scheduler.Context
+		backupLocationUID          string
+		cloudCredUID               string
+		bkpNamespaces              []string
+		clusterUid                 string
+		backupName                 string
+		credName                   string
+		cloudCredUidList           []string
+		backupLocationName         string
+		nsLabels                   map[string]string
+		randomLabel                string
+		restoreName                string
+		periodicSchedulePolicyName string
+		periodicSchedulePolicyUid  string
+		restoreNames               []string
+		firstScheduleBackupName    string
+		scheduleNames              []string
+		scheduleBackupName         string
+		labeledSelector            map[string]string
+	)
+	labelSelectors := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("BasicBackupAndScheduleBackupUsingNamespaceLabel", "Schedule backup and restore, using namespace label and resource label", nil, 58041)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
```

Comment:
Change it to the number of namespaces the test case needs

Reply:
Done